src/components/QuizTemplate.tsx

```diff
@@ -17,7 +31,10 @@ interface QuizProps {
 }
 
 const QuizTemplate: React.FC<QuizProps> = (QuizProps) => {
-  const [quiz, setQuiz] = useState(fullQuiz);
+  const [quiz, setQuiz] = useState([...cssQuiz, ...freecodecampQuiz, 
+                                    ...generalCSQuiz, ...gitQuiz, ...htmlQuiz, ...informationTechnologyQuiz, 
+                                    ...javascriptQuiz, ...linuxQuiz, ...pythonQuiz, ...sqlQuiz,
+                                    ...agileQuiz, ...qualityAssuranceQuiz, ...securityQuiz]);
```

Comment:
Let's create a variable here for all of the quiz categories. That way we only need to update it in one spot and it lessens the chance for someone in the future forgetting to add all of the categories in the `setQuiz `

```js
const allCategories = [...cssQuiz, ...freecodecampQuiz, 
                                    ...generalCSQuiz, ...gitQuiz, ...htmlQuiz, ...informationTechnologyQuiz, 
                                    ...javascriptQuiz, ...linuxQuiz, ...pythonQuiz, ...sqlQuiz,
                                    ...agileQuiz, ...qualityAssuranceQuiz, ...securityQuiz]
const [quiz, setQuiz] = useState(allCategories)
```